host/windows/syscall.c

```diff
@@ -723,5 +1181,25 @@ int oe_syscall_getgroups_ocall(size_t size, unsigned int* list)
 
 int oe_syscall_uname_ocall(struct oe_utsname* buf)
 {
-    PANIC;
+    // Based on
+    // https://docs.microsoft.com/en-us/windows/win32/sysinfo/getting-the-system-version
+    // OE SDK is supported only on WindowsServer and Win10
+    if (IsWindowsServer())
+    {
+        sprintf(buf->nodename, "(unknown)");
```

Comment:
nodename should be set to whatever gethostname() returns

Reply:
Used GetComputerNameEx